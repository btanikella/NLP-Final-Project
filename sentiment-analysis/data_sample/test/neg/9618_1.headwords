done
drunk
looks
seconds
save
remember
Also
camera
