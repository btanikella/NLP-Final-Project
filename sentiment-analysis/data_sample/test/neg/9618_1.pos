DT NN NN VBN IN NNP NNP VBD RB RB VBN
DT VBG NN MD VB VBN CC JJ  JJ CC RB RB IN DT NN IN PRP VBD VBG VBN
PRP VBZ IN PRP VBD VBN IN IN DT NN NN NN CC IN RB IN DT NN  DT JJ NN VBZ
DT NN VBZ IN CD CC CD NNS IN DT NN
IN PRP VBP VBG IN VBG DT NN  VB VB PRP$ NN
PRP VBP VBG DT NN IN DT NN RB IN CD
RB  DT NNP NN IN DT NN IN DT NN IN  WRB NNP VBZ  VBZ JJR IN DT NN  CC DT VBG JJ NNS IN DT NN VBP  VBG
WRB VBZ DT JJ NN JJ
