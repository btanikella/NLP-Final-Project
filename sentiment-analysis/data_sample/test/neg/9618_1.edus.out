Sentence #1 (12 tokens):
The DVD version released by Crash Cinema was very poorly done.
[Text=The CharacterOffsetBegin=4 CharacterOffsetEnd=7 PartOfSpeech=DT] [Text=DVD CharacterOffsetBegin=8 CharacterOffsetEnd=11 PartOfSpeech=NN] [Text=version CharacterOffsetBegin=12 CharacterOffsetEnd=19 PartOfSpeech=NN] [Text=released CharacterOffsetBegin=20 CharacterOffsetEnd=28 PartOfSpeech=VBN] [Text=by CharacterOffsetBegin=29 CharacterOffsetEnd=31 PartOfSpeech=IN] [Text=Crash CharacterOffsetBegin=32 CharacterOffsetEnd=37 PartOfSpeech=NNP] [Text=Cinema CharacterOffsetBegin=38 CharacterOffsetEnd=44 PartOfSpeech=NNP] [Text=was CharacterOffsetBegin=45 CharacterOffsetEnd=48 PartOfSpeech=VBD] [Text=very CharacterOffsetBegin=49 CharacterOffsetEnd=53 PartOfSpeech=RB] [Text=poorly CharacterOffsetBegin=54 CharacterOffsetEnd=60 PartOfSpeech=RB] [Text=done CharacterOffsetBegin=61 CharacterOffsetEnd=65 PartOfSpeech=VBN] [Text=. CharacterOffsetBegin=65 CharacterOffsetEnd=66 PartOfSpeech=.] 
root(ROOT-0, done-11)
det(version-3, The-1)
nn(version-3, DVD-2)
nsubjpass(done-11, version-3)
partmod(version-3, released-4)
prep(released-4, by-5)
nn(Cinema-7, Crash-6)
pobj(by-5, Cinema-7)
auxpass(done-11, was-8)
advmod(poorly-10, very-9)
advmod(done-11, poorly-10)
punct(done-11, .-12)

Sentence #2 (22 tokens):
The mastering engineer must have been either drunk, asleep or not even in the room while it was being done.
[Text=The CharacterOffsetBegin=71 CharacterOffsetEnd=74 PartOfSpeech=DT] [Text=mastering CharacterOffsetBegin=75 CharacterOffsetEnd=84 PartOfSpeech=VBG] [Text=engineer CharacterOffsetBegin=85 CharacterOffsetEnd=93 PartOfSpeech=NN] [Text=must CharacterOffsetBegin=94 CharacterOffsetEnd=98 PartOfSpeech=MD] [Text=have CharacterOffsetBegin=99 CharacterOffsetEnd=103 PartOfSpeech=VB] [Text=been CharacterOffsetBegin=104 CharacterOffsetEnd=108 PartOfSpeech=VBN] [Text=either CharacterOffsetBegin=109 CharacterOffsetEnd=115 PartOfSpeech=CC] [Text=drunk CharacterOffsetBegin=116 CharacterOffsetEnd=121 PartOfSpeech=JJ] [Text=, CharacterOffsetBegin=121 CharacterOffsetEnd=122 PartOfSpeech=,] [Text=asleep CharacterOffsetBegin=123 CharacterOffsetEnd=129 PartOfSpeech=JJ] [Text=or CharacterOffsetBegin=130 CharacterOffsetEnd=132 PartOfSpeech=CC] [Text=not CharacterOffsetBegin=133 CharacterOffsetEnd=136 PartOfSpeech=RB] [Text=even CharacterOffsetBegin=137 CharacterOffsetEnd=141 PartOfSpeech=RB] [Text=in CharacterOffsetBegin=142 CharacterOffsetEnd=144 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=145 CharacterOffsetEnd=148 PartOfSpeech=DT] [Text=room CharacterOffsetBegin=149 CharacterOffsetEnd=153 PartOfSpeech=NN] [Text=while CharacterOffsetBegin=154 CharacterOffsetEnd=159 PartOfSpeech=IN] [Text=it CharacterOffsetBegin=160 CharacterOffsetEnd=162 PartOfSpeech=PRP] [Text=was CharacterOffsetBegin=163 CharacterOffsetEnd=166 PartOfSpeech=VBD] [Text=being CharacterOffsetBegin=167 CharacterOffsetEnd=172 PartOfSpeech=VBG] [Text=done CharacterOffsetBegin=173 CharacterOffsetEnd=177 PartOfSpeech=VBN] [Text=. CharacterOffsetBegin=177 CharacterOffsetEnd=178 PartOfSpeech=.] 
root(ROOT-0, drunk-8)
det(engineer-3, The-1)
amod(engineer-3, mastering-2)
nsubj(drunk-8, engineer-3)
aux(drunk-8, must-4)
aux(drunk-8, have-5)
cop(drunk-8, been-6)
preconj(drunk-8, either-7)
punct(drunk-8, ,-9)
appos(drunk-8, asleep-10)
cc(asleep-10, or-11)
conj(asleep-10, not-12)
advmod(asleep-10, even-13)
prep(asleep-10, in-14)
det(room-16, the-15)
pobj(in-14, room-16)
mark(done-21, while-17)
nsubjpass(done-21, it-18)
aux(done-21, was-19)
auxpass(done-21, being-20)
advcl(asleep-10, done-21)
punct(drunk-8, .-22)

Sentence #3 (24 tokens):
It looks like it was mastered from about a tenth generation copy and about halfway through the film, the audio synchronization disappears.
[Text=It CharacterOffsetBegin=183 CharacterOffsetEnd=185 PartOfSpeech=PRP] [Text=looks CharacterOffsetBegin=186 CharacterOffsetEnd=191 PartOfSpeech=VBZ] [Text=like CharacterOffsetBegin=192 CharacterOffsetEnd=196 PartOfSpeech=IN] [Text=it CharacterOffsetBegin=197 CharacterOffsetEnd=199 PartOfSpeech=PRP] [Text=was CharacterOffsetBegin=200 CharacterOffsetEnd=203 PartOfSpeech=VBD] [Text=mastered CharacterOffsetBegin=204 CharacterOffsetEnd=212 PartOfSpeech=VBN] [Text=from CharacterOffsetBegin=213 CharacterOffsetEnd=217 PartOfSpeech=IN] [Text=about CharacterOffsetBegin=218 CharacterOffsetEnd=223 PartOfSpeech=IN] [Text=a CharacterOffsetBegin=224 CharacterOffsetEnd=225 PartOfSpeech=DT] [Text=tenth CharacterOffsetBegin=226 CharacterOffsetEnd=231 PartOfSpeech=NN] [Text=generation CharacterOffsetBegin=232 CharacterOffsetEnd=242 PartOfSpeech=NN] [Text=copy CharacterOffsetBegin=243 CharacterOffsetEnd=247 PartOfSpeech=NN] [Text=and CharacterOffsetBegin=248 CharacterOffsetEnd=251 PartOfSpeech=CC] [Text=about CharacterOffsetBegin=252 CharacterOffsetEnd=257 PartOfSpeech=IN] [Text=halfway CharacterOffsetBegin=258 CharacterOffsetEnd=265 PartOfSpeech=RB] [Text=through CharacterOffsetBegin=266 CharacterOffsetEnd=273 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=274 CharacterOffsetEnd=277 PartOfSpeech=DT] [Text=film CharacterOffsetBegin=278 CharacterOffsetEnd=282 PartOfSpeech=NN] [Text=, CharacterOffsetBegin=282 CharacterOffsetEnd=283 PartOfSpeech=,] [Text=the CharacterOffsetBegin=284 CharacterOffsetEnd=287 PartOfSpeech=DT] [Text=audio CharacterOffsetBegin=288 CharacterOffsetEnd=293 PartOfSpeech=JJ] [Text=synchronization CharacterOffsetBegin=294 CharacterOffsetEnd=309 PartOfSpeech=NN] [Text=disappears CharacterOffsetBegin=310 CharacterOffsetEnd=320 PartOfSpeech=VBZ] [Text=. CharacterOffsetBegin=320 CharacterOffsetEnd=321 PartOfSpeech=.] 
root(ROOT-0, looks-2)
nsubj(looks-2, It-1)
mark(mastered-6, like-3)
nsubjpass(mastered-6, it-4)
auxpass(mastered-6, was-5)
advcl(looks-2, mastered-6)
prep(mastered-6, from-7)
pcomp(from-7, about-8)
det(copy-12, a-9)
amod(copy-12, tenth-10)
nn(copy-12, generation-11)
pobj(about-8, copy-12)
cc(about-8, and-13)
conj(about-8, about-14)
pobj(about-14, halfway-15)
prep(halfway-15, through-16)
det(film-18, the-17)
pobj(through-16, film-18)
punct(looks-2, ,-19)
det(synchronization-22, the-20)
amod(synchronization-22, audio-21)
nsubj(disappears-23, synchronization-22)
ccomp(looks-2, disappears-23)
punct(looks-2, .-24)

Sentence #4 (12 tokens):
The dialog is about 10 or 15 seconds behind the audio.
[Text=The CharacterOffsetBegin=326 CharacterOffsetEnd=329 PartOfSpeech=DT] [Text=dialog CharacterOffsetBegin=330 CharacterOffsetEnd=336 PartOfSpeech=NN] [Text=is CharacterOffsetBegin=337 CharacterOffsetEnd=339 PartOfSpeech=VBZ] [Text=about CharacterOffsetBegin=340 CharacterOffsetEnd=345 PartOfSpeech=IN] [Text=10 CharacterOffsetBegin=346 CharacterOffsetEnd=348 PartOfSpeech=CD] [Text=or CharacterOffsetBegin=349 CharacterOffsetEnd=351 PartOfSpeech=CC] [Text=15 CharacterOffsetBegin=352 CharacterOffsetEnd=354 PartOfSpeech=CD] [Text=seconds CharacterOffsetBegin=355 CharacterOffsetEnd=362 PartOfSpeech=NNS] [Text=behind CharacterOffsetBegin=363 CharacterOffsetEnd=369 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=370 CharacterOffsetEnd=373 PartOfSpeech=DT] [Text=audio CharacterOffsetBegin=374 CharacterOffsetEnd=379 PartOfSpeech=NN] [Text=. CharacterOffsetBegin=379 CharacterOffsetEnd=380 PartOfSpeech=.] 
root(ROOT-0, seconds-8)
det(dialog-2, The-1)
nsubj(seconds-8, dialog-2)
cop(seconds-8, is-3)
amod(seconds-8, about-4)
num(seconds-8, 10-5)
cc(10-5, or-6)
conj(10-5, 15-7)
prep(seconds-8, behind-9)
det(audio-11, the-10)
pobj(behind-9, audio-11)
punct(seconds-8, .-12)

Sentence #5 (14 tokens):
If you're thinking about purchasing this DVD, please save your money.
[Text=If CharacterOffsetBegin=385 CharacterOffsetEnd=387 PartOfSpeech=IN] [Text=you CharacterOffsetBegin=388 CharacterOffsetEnd=391 PartOfSpeech=PRP] [Text='re CharacterOffsetBegin=391 CharacterOffsetEnd=394 PartOfSpeech=VBP] [Text=thinking CharacterOffsetBegin=395 CharacterOffsetEnd=403 PartOfSpeech=VBG] [Text=about CharacterOffsetBegin=404 CharacterOffsetEnd=409 PartOfSpeech=IN] [Text=purchasing CharacterOffsetBegin=410 CharacterOffsetEnd=420 PartOfSpeech=VBG] [Text=this CharacterOffsetBegin=421 CharacterOffsetEnd=425 PartOfSpeech=DT] [Text=DVD CharacterOffsetBegin=426 CharacterOffsetEnd=429 PartOfSpeech=NN] [Text=, CharacterOffsetBegin=429 CharacterOffsetEnd=430 PartOfSpeech=,] [Text=please CharacterOffsetBegin=431 CharacterOffsetEnd=437 PartOfSpeech=VB] [Text=save CharacterOffsetBegin=438 CharacterOffsetEnd=442 PartOfSpeech=VB] [Text=your CharacterOffsetBegin=443 CharacterOffsetEnd=447 PartOfSpeech=PRP$] [Text=money CharacterOffsetBegin=448 CharacterOffsetEnd=453 PartOfSpeech=NN] [Text=. CharacterOffsetBegin=453 CharacterOffsetEnd=454 PartOfSpeech=.] 
root(ROOT-0, save-11)
mark(thinking-4, If-1)
nsubj(thinking-4, you-2)
aux(thinking-4, 're-3)
advcl(save-11, thinking-4)
prep(thinking-4, about-5)
pcomp(about-5, purchasing-6)
det(DVD-8, this-7)
dobj(purchasing-6, DVD-8)
punct(save-11, ,-9)
aux(save-11, please-10)
poss(money-13, your-12)
dobj(save-11, money-13)
punct(save-11, .-14)

Sentence #6 (12 tokens):
I remember seeing this film at the theater back in 1973.
[Text=I CharacterOffsetBegin=459 CharacterOffsetEnd=460 PartOfSpeech=PRP] [Text=remember CharacterOffsetBegin=461 CharacterOffsetEnd=469 PartOfSpeech=VBP] [Text=seeing CharacterOffsetBegin=470 CharacterOffsetEnd=476 PartOfSpeech=VBG] [Text=this CharacterOffsetBegin=477 CharacterOffsetEnd=481 PartOfSpeech=DT] [Text=film CharacterOffsetBegin=482 CharacterOffsetEnd=486 PartOfSpeech=NN] [Text=at CharacterOffsetBegin=487 CharacterOffsetEnd=489 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=490 CharacterOffsetEnd=493 PartOfSpeech=DT] [Text=theater CharacterOffsetBegin=494 CharacterOffsetEnd=501 PartOfSpeech=NN] [Text=back CharacterOffsetBegin=502 CharacterOffsetEnd=506 PartOfSpeech=RB] [Text=in CharacterOffsetBegin=507 CharacterOffsetEnd=509 PartOfSpeech=IN] [Text=1973 CharacterOffsetBegin=510 CharacterOffsetEnd=514 PartOfSpeech=CD] [Text=. CharacterOffsetBegin=514 CharacterOffsetEnd=515 PartOfSpeech=.] 
root(ROOT-0, remember-2)
nsubj(remember-2, I-1)
xcomp(remember-2, seeing-3)
det(film-5, this-4)
dobj(seeing-3, film-5)
prep(seeing-3, at-6)
det(theater-8, the-7)
pobj(at-6, theater-8)
advmod(seeing-3, back-9)
prep(back-9, in-10)
pobj(in-10, 1973-11)
punct(remember-2, .-12)

Sentence #7 (36 tokens):
Also, the VHS copy of this film under the title of "When Taekwondo Strikes" looks better than the DVD, but the remaining several minutes of the movie are "missing".
[Text=Also CharacterOffsetBegin=520 CharacterOffsetEnd=524 PartOfSpeech=RB] [Text=, CharacterOffsetBegin=524 CharacterOffsetEnd=525 PartOfSpeech=,] [Text=the CharacterOffsetBegin=526 CharacterOffsetEnd=529 PartOfSpeech=DT] [Text=VHS CharacterOffsetBegin=530 CharacterOffsetEnd=533 PartOfSpeech=NNP] [Text=copy CharacterOffsetBegin=534 CharacterOffsetEnd=538 PartOfSpeech=NN] [Text=of CharacterOffsetBegin=539 CharacterOffsetEnd=541 PartOfSpeech=IN] [Text=this CharacterOffsetBegin=542 CharacterOffsetEnd=546 PartOfSpeech=DT] [Text=film CharacterOffsetBegin=547 CharacterOffsetEnd=551 PartOfSpeech=NN] [Text=under CharacterOffsetBegin=552 CharacterOffsetEnd=557 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=558 CharacterOffsetEnd=561 PartOfSpeech=DT] [Text=title CharacterOffsetBegin=562 CharacterOffsetEnd=567 PartOfSpeech=NN] [Text=of CharacterOffsetBegin=568 CharacterOffsetEnd=570 PartOfSpeech=IN] [Text=`` CharacterOffsetBegin=571 CharacterOffsetEnd=572 PartOfSpeech=``] [Text=When CharacterOffsetBegin=572 CharacterOffsetEnd=576 PartOfSpeech=WRB] [Text=Taekwondo CharacterOffsetBegin=577 CharacterOffsetEnd=586 PartOfSpeech=NNP] [Text=Strikes CharacterOffsetBegin=587 CharacterOffsetEnd=594 PartOfSpeech=VBZ] [Text='' CharacterOffsetBegin=594 CharacterOffsetEnd=595 PartOfSpeech=''] [Text=looks CharacterOffsetBegin=596 CharacterOffsetEnd=601 PartOfSpeech=VBZ] [Text=better CharacterOffsetBegin=602 CharacterOffsetEnd=608 PartOfSpeech=JJR] [Text=than CharacterOffsetBegin=609 CharacterOffsetEnd=613 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=614 CharacterOffsetEnd=617 PartOfSpeech=DT] [Text=DVD CharacterOffsetBegin=618 CharacterOffsetEnd=621 PartOfSpeech=NN] [Text=, CharacterOffsetBegin=621 CharacterOffsetEnd=622 PartOfSpeech=,] [Text=but CharacterOffsetBegin=623 CharacterOffsetEnd=626 PartOfSpeech=CC] [Text=the CharacterOffsetBegin=627 CharacterOffsetEnd=630 PartOfSpeech=DT] [Text=remaining CharacterOffsetBegin=631 CharacterOffsetEnd=640 PartOfSpeech=VBG] [Text=several CharacterOffsetBegin=641 CharacterOffsetEnd=648 PartOfSpeech=JJ] [Text=minutes CharacterOffsetBegin=649 CharacterOffsetEnd=656 PartOfSpeech=NNS] [Text=of CharacterOffsetBegin=657 CharacterOffsetEnd=659 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=660 CharacterOffsetEnd=663 PartOfSpeech=DT] [Text=movie CharacterOffsetBegin=664 CharacterOffsetEnd=669 PartOfSpeech=NN] [Text=are CharacterOffsetBegin=670 CharacterOffsetEnd=673 PartOfSpeech=VBP] [Text=`` CharacterOffsetBegin=674 CharacterOffsetEnd=675 PartOfSpeech=``] [Text=missing CharacterOffsetBegin=675 CharacterOffsetEnd=682 PartOfSpeech=VBG] [Text='' CharacterOffsetBegin=682 CharacterOffsetEnd=683 PartOfSpeech=''] [Text=. CharacterOffsetBegin=683 CharacterOffsetEnd=684 PartOfSpeech=.] 
root(ROOT-0, Also-1)
punct(Also-1, ,-2)
det(copy-5, the-3)
nn(copy-5, VHS-4)
nsubj(,-2, copy-5)
prep(copy-5, of-6)
det(film-8, this-7)
pobj(of-6, film-8)
prep(copy-5, under-9)
det(title-11, the-10)
pobj(under-9, title-11)
prep(title-11, of-12)
punct(of-12, ``-13)
advmod(Strikes-16, When-14)
nsubj(Strikes-16, Taekwondo-15)
dep(of-12, Strikes-16)
punct(Strikes-16, ''-17)
ccomp(Strikes-16, looks-18)
acomp(looks-18, better-19)
prep(better-19, than-20)
det(DVD-22, the-21)
pobj(than-20, DVD-22)
punct(Strikes-16, ,-23)
cc(Strikes-16, but-24)
det(minutes-28, the-25)
amod(minutes-28, remaining-26)
amod(minutes-28, several-27)
nsubj(missing-34, minutes-28)
prep(minutes-28, of-29)
det(movie-31, the-30)
pobj(of-29, movie-31)
aux(missing-34, are-32)
punct(missing-34, ``-33)
conj(Strikes-16, missing-34)
punct(missing-34, ''-35)
punct(copy-5, .-36)

Sentence #8 (7 tokens):
Where is the original camera negative?
[Text=Where CharacterOffsetBegin=689 CharacterOffsetEnd=694 PartOfSpeech=WRB] [Text=is CharacterOffsetBegin=695 CharacterOffsetEnd=697 PartOfSpeech=VBZ] [Text=the CharacterOffsetBegin=698 CharacterOffsetEnd=701 PartOfSpeech=DT] [Text=original CharacterOffsetBegin=702 CharacterOffsetEnd=710 PartOfSpeech=JJ] [Text=camera CharacterOffsetBegin=711 CharacterOffsetEnd=717 PartOfSpeech=NN] [Text=negative CharacterOffsetBegin=718 CharacterOffsetEnd=726 PartOfSpeech=JJ] [Text=? CharacterOffsetBegin=726 CharacterOffsetEnd=727 PartOfSpeech=.] 
root(ROOT-0, camera-5)
advmod(camera-5, Where-1)
cop(camera-5, is-2)
det(camera-5, the-3)
amod(camera-5, original-4)
amod(camera-5, negative-6)
punct(camera-5, ?-7)

